src/torchmetrics/__init__.py

```diff
@@ -14,9 +14,6 @@
 _PACKAGE_ROOT = os.path.dirname(__file__)
 _PROJECT_ROOT = os.path.dirname(_PACKAGE_ROOT)
 
-if package_available("pretty_errors"):
```

Comment:
`pretty_errors` was the only package listed in `debug.txt`, so I wasn't sure if it would be better to delete the whole file or just the one line.  I decided to be conservative and delete the line.  Let me know if that was the wrong decision.

Reply:
Let's keep it empty